src/Test/L0/TestHostContext.cs

```diff
@@ -103,8 +103,8 @@ public async Task Delay(TimeSpan delay, CancellationToken token)
                 handler(this, new DelayEventArgs(delay, token));
             }
 
-            // Delay zero
-            await Task.Delay(TimeSpan.Zero);
```

Comment:
`zero`  will make the test case stuck in case we use the HostContext.Delay in some background task.